Api/Statistics.php

```diff
@@ -248,7 +253,12 @@ public function getStatsByPlatform($platformId, \DateTime $beginDate = null, \Da
         ];
 
         $response = $this->httpClient->createAndSendRequest($args);
-        $data     = $this->processResponse($response);
+
+        if (null !== $this->logger) {
+            $this->logger->info('Response : '.$response);
```

Comment:
Tes logs ne sont pas contextualisés, ils vont se perdre dans le flot des logs de l'app
```suggestion
            $this->logger->info('WannaSpeak Response : '.$response);
```